WBooks/BookList/BookList.swift

```diff
@@ -11,19 +11,39 @@ class BookList: UIViewController, UITableViewDataSource, UITableViewDelegate {
     var books: [Book] = []
     @IBOutlet weak var booksTable: UITableView!
     
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         books = getBooks()
         booksTable.dataSource = self
         booksTable.delegate = self
         let nib = UINib(nibName: BookListCellTableViewCell.identifier, bundle: Bundle.main)
         booksTable.register(nib, forCellReuseIdentifier: BookListCellTableViewCell.identifier)
```

Comment:
These can go in a separate `setUpView` method so the lifecycle methods (`loadView`, `viewDidLoad`, etc.) are not too cluttered.